### PR TITLE
Issue/#69 setup test framework

### PR DESCRIPTION
**Pull request by Alessandro100**
# Related Issue
https://github.com/DPigeon/Money-Tree/issues/69

# Proposed changes
- Removed Karma and replaced it with Jest
- Added coverage report in Spock

# Additional Info
- Information of tests added to README

# Checklist (if applicable)
- N/A Tests
- [X] Documentation

# Screenshots (if applicable)

| Before 🐛 | After 🦋 |
| --- | --- |
|  |  |

# Video or links (if applicable)
 - NA

# Reviewer(s)
 - @razine-bensari 
- @Alessandro100 
- @DPigeon

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## JAVA
 
-You need java 11 to run the application. You can check you java version by running this command in your terminal: `java -version`.
```
Comment (Alessandro100) [inline backend/README.md:5]: @razine-bensari how come you removed this?

File: backend/README.md [hunk 2]
```diff
@@ -25,19 +25,24 @@ to connect if the password has not been changed.
 
 - To start neo4j locally, open your terminal and run `neo4j start`
 
-- If neo4j is up and running locally, you can run `curl -v -u neo4j:neo4j -X POST localhost:7474/user/neo4j/password -H "Content-type:application/json" -d "{\"password\":\"stonecap\"}"` to change the default password to stonecap. This our db password for dev when running the application locally.
-If this does not work, you can go through the browser at localhost:7474 and change the password from there.
+- If neo4j is up and running locally, you can run `curl -v -u neo4j:neo4j -X POST localhost:7474/user/neo4j/password -H "Content-type:application/json" -d "{\"password\":\"secret\"}"` and replace the secret string with stonecap. stonecap is our db password for dev when running the application locally.
```
Comment (Alessandro100) [inline backend/README.md:28]: @razine-bensari why did you change this?

**Review by razine-bensari — APPROVED** [review]

File: backend/README.md (hunk 1 above)
Comment (razine-bensari) [reply backend/README.md:5]: fixed

**Review by razine-bensari — APPROVED** [review]

**Comment by DPigeon** [comment]
Good job! Just tested and works fine.
